platform/app/public/config/default.js

```diff
@@ -21,6 +21,7 @@ window.config = {
   },
   // filterQueryParam: false,
   defaultDataSourceName: 'dicomweb',
+  showPatientInfo: true,
```

Comment:
This should have mor options, should be
- `visibleCollapsed`: shows but collapsed (our default) no need to put here

Documented
- `disabled`: does not even show anything
- `visible`: visible (not collapsed)